modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -181,12 +181,12 @@ $ oc get csv -n {CNVNamespace}
 +
 The update completes when the `VERSION` field matches the target EUS version and the `PHASE` field reads `Succeeded`.
 
-. Restore the workload update methods configuration that you backed up:
+. Restore the `workloadUpdateMethods` configuration that you recorded earlier with the following command:
```

Comment:
```suggestion
. Restore the `workloadUpdateMethods` configuration you recorded from step 1 with the following command:
```
Just a suggestion, but I think referring to the step as opposed to "from earlier" would be clearer here.